### PR TITLE
Langflow 1.0 docs

### PR DESCRIPTION
**Pull request by mendonk**
Docs final updates still in progress.

### CODE REVIEW

File: docs/docs/getting-started/workspace.mdx [hunk 1]
```diff
@@ -4,56 +4,107 @@ import ZoomableImage from "/src/theme/ZoomableImage.js";
...
 />
 
-In this flow, the **OpenAI LLM component** receives input (left side) and produces output (right side) - in this case, receiving input from the **Chat Input** and **Prompt** components and producing output to the **Chat Output** component.
+In this flow, the **OpenAI Model** receives input (left side) and produces output (right side) - in this case, receiving input from the **Chat Input** and **Prompt** components and passing the output to the **Chat Output** component.
 
 ## Component
 
-Components are the building blocks of flows. They consist of inputs, outputs, and parameters that define their functionality. These elements provide a convenient and straightforward way to compose LLM-based applications. Learn more about components and how they work in the LangChain [documentation](https://python.langchain.com/docs/integrations/components).
+Components are the building blocks of flows. They consist of inputs, outputs, and parameters that define their functionality. These elements provide a convenient and straightforward way to compose LLM-based applications. Learn more about components and how they work in the [Components Section](../components).
```
Comment (ogabrielluiz) [inline docs/docs/getting-started/workspace.mdx:95]: This reference is raising a build error.

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Thank you, @mendonk !

File: docs/docs/contributing/contribute-component.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ import ZoomableImage from "/src/theme/ZoomableImage.js";
 
 # How to Contribute Components?
 
-As of Langflow 1.0 alpha, new components are added as objects of the [CustomComponent](https://github.com/langflow-ai/langflow/blob/dev/src/backend/base/langflow/interface/custom/custom_component/custom_component.py) class and any dependencies are added to the [pyproject.toml](https://github.com/langflow-ai/langflow/blob/dev/pyproject.toml#L27) file.
+New components are added as objects of the [CustomComponent](https://github.com/langflow-ai/langflow/blob/dev/src/backend/base/langflow/interface/custom/custom_component/custom_component.py) class and any dependencies are added to the [pyproject.toml](https://github.com/langflow-ai/langflow/blob/dev/pyproject.toml#L27) file.
```
Comment (ogabrielluiz) [inline docs/docs/contributing/contribute-component.md:5]: This should be updated to `Component` which is at `src/backend/base/langflow/custom/custom_component/component.py`

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
There's this little change about the CustomComponent and the other thing I think we need is an explanation of each field type. 

The MessageInput is very similar to the TextInput. The only difference is that the TextInput will automatically try to get the `text` value from a Message.

**Review by ogabrielluiz — APPROVED** [review]
LGTM

File: docs/docs/getting-started/workspace.mdx (hunk 1 above)
Comment (mendonk) [reply docs/docs/getting-started/workspace.mdx]: Thanks for fixing this @ogabrielluiz

**Pull request merged**